### PR TITLE
test(robot): add basic v2 volume test

### PR DESCRIPTION
**Pull request by yangchiu**
#### Which issue(s) this PR fixes:

Issue https://github.com/longhorn/longhorn/issues/8259

#### What this PR does / why we need it:

add basic v2 volume test

### CODE REVIEW

File: e2e/libs/setting/setting.py [hunk 1]
```diff
@@ -1,73 +1,57 @@
...
-from setting.constant import SETTING_BACKUP_TARGET
-from setting.constant import SETTING_BACKUP_TARGET_CREDENTIAL_SECRET
-from setting.constant import SETTING_BACKUPSTORE_POLL_INTERVAL
+from utility.utility import get_retry_count_and_interval
 
 class Setting:
 
+    SETTING_BACKUP_TARGET = "backup-target"
+    SETTING_BACKUP_TARGET_CREDENTIAL_SECRET = "backup-target-credential-secret"
+    SETTING_BACKUPSTORE_POLL_INTERVAL = "backupstore-poll-interval"
```
Comment (c3y1huang) [inline e2e/libs/setting/setting.py:11]: It seems that you're deleting the setting/constant.py file. Is there a conversation happening about this change?

Comment (yangchiu) [reply e2e/libs/setting/setting.py]: Look like `setting` didn't change in this PR. An additional ticket can be created if this modification is required.

**Review by khushboo-rancher — CHANGES_REQUESTED** [review]
I feel, we can have a directory where all the test cases related to v2 can be implemented. In future, we'll have many test cases for v2, we can structure something them similar to v1 test cases.

**Comment by yangchiu** [comment]
> I feel, we can have a directory where all the test cases related to v2 can be implemented. In future, we'll have many test cases for v2, we can structure something them similar to v1 test cases.

Refined. Thank you!

**Review by khushboo-rancher — APPROVED** [review]

**Pull request merged**